src/pages/workspace/upgrade/UpgradeIntro.tsx

```diff
@@ -96,7 +98,7 @@ function UpgradeIntro({feature, onUpgrade, buttonDisabled, loading, isCategorizi
                     <Text style={[styles.textNormal, styles.textSupporting, styles.mb4]}>{translate(feature.description)}</Text>
                     <Text style={[styles.textNormal, styles.textSupporting]}>
                         {translate(`workspace.upgrade.${feature.id}.onlyAvailableOnPlan`)}
-                        <Text style={[styles.textSupporting, styles.textBold]}>{formattedPrice}</Text>
+                        <Text style={[styles.textSupporting, styles.textBold]}>formattedPrice</Text>
```

Comment:
This doesn't look right

<img width="336" alt="image" src="https://github.com/user-attachments/assets/660f4c3b-6707-4a2d-a165-1002a2637d5c" />